chef-universal-mingw-ucrt.gemspec

```diff
@@ -2,22 +2,22 @@ gemspec = instance_eval(File.read(File.expand_path("chef.gemspec", __dir__)))
 
 gemspec.platform = Gem::Platform.new(%w{x64-mingw-ucrt})
 
-# gemspec.add_dependency "win32-api", "~> 1.10.0"
-# gemspec.add_dependency "win32-event", "~> 0.6.1"
+gemspec.add_dependency "win32-api", "~> 1.10.0"
+gemspec.add_dependency "win32-event", "~> 0.6.1"
 # TODO: Relax this pin and make the necessary updaets. The issue originally
 # leading to this pin has been fixed in 0.6.5.
-# gemspec.add_dependency "win32-eventlog", "0.6.3"
-# gemspec.add_dependency "win32-mmap", "~> 0.4.1"
-# gemspec.add_dependency "win32-mutex", "~> 0.4.2"
-# gemspec.add_dependency "win32-process", "~> 0.9"
-# gemspec.add_dependency "win32-service", ">= 2.1.5", "< 3.0"
-# gemspec.add_dependency "wmi-lite", "~> 1.0"
-# gemspec.add_dependency "win32-taskscheduler", "~> 2.0"
-# gemspec.add_dependency "iso8601", ">= 0.12.1", "< 0.14" # validate 0.14 when it comes out
-# gemspec.add_dependency "win32-certstore", "~> 0.6.15" # 0.5+ required for specifying user vs. system store
-# gemspec.add_dependency "chef-powershell", "~> 1.0.12" # The guts of the powershell_exec code have been moved to its own gem, chef-powershell. It's part of the chef-powershell-shim repo.
+gemspec.add_dependency "win32-eventlog", "0.6.3"
+gemspec.add_dependency "win32-mmap", "~> 0.4.1"
+gemspec.add_dependency "win32-mutex", "~> 0.4.2"
+gemspec.add_dependency "win32-process", "~> 0.9"
+gemspec.add_dependency "win32-service", ">= 2.1.5", "< 3.0"
+gemspec.add_dependency "wmi-lite", "~> 1.0"
+gemspec.add_dependency "win32-taskscheduler", "~> 2.0"
+gemspec.add_dependency "iso8601", ">= 0.12.1", "< 0.14" # validate 0.14 when it comes out
+gemspec.add_dependency "win32-certstore", "~> 0.6.15" # 0.5+ required for specifying user vs. system store
+gemspec.add_dependency "chef-powershell", "~> 1.0.12" # The guts of the powershell_exec code have been moved to its own gem, chef-powershell. It's part of the chef-powershell-shim repo.
+
 gemspec.extensions << "ext/win32-eventlog/Rakefile"
 gemspec.files += Dir.glob("{distro,ext}/**/*")
-gemspec.required_ruby_version = ">= 3.1.2"
```

Comment:
This differs from required_ruby_version of 3.1.0 in chef.gemspec; just wanted to confirm that 3.1.0 is ok, and it shouldn't be 3.1.2 there.

Reply:
Added to chef.gemspec

Reply:
We have reverted it back since we started seeing failures on verify pipeline due to boxes being configured for ruby 3.1.1